src/doc/man/cargo-vendor.md

```diff
@@ -27,7 +27,7 @@ to use the vendored sources, which you will need to add to `.cargo/config.toml`.
 
 {{#option "`-s` _manifest_" "`--sync` _manifest_" }}
 Specify extra `Cargo.toml` manifests to workspaces which should also be
-vendored and synced to the output.
+vendored and synced to the output. Terminate the list with `--`.
```

Comment:
Yes, getting rid of `multiple_values` is definitely the right thing to do, but I didn't feel comfortable making a UI-incompatible change. Best I could think of was documenting it.

I didn't realize the terminator killed parsing entirely. Is that also true if we specify `value_terminator` in the declaration for the `--sync` option?

Reply:
Oh dear yeah it was never intended that `--foo bar baz` is considered for Cargo, that's definitely an antipattern we wouldn't ever actually opt-in to. I would hope that `cargo vendor` isn't so widely used we could just fix it and consider it a bugfix

Reply:
If you're OK with changing the UI, then I've opened #10448 to do that.

Reply:
I briefly reviewed all other usages of `multiple_values` and they all look OK to me (mostly argument lists, some spec lists, but nothing like cargo-vendor where it could eat a following significant argument).